modules/virtualnetwork/variables.tf

```diff
@@ -65,6 +65,11 @@ variable "virtual_networks" {
     vwan_hub_resource_id                     = optional(string, "")
     vwan_propagated_routetables_labels       = optional(list(string), [])
     vwan_propagated_routetables_resource_ids = optional(list(string), [])
+    vwan_security_configuration = object({
```

Comment:
please can you wrap the vwan_security_configuration in an `optional()` with an empty object as default?

Reply:
Just getting to this, it's a nested object within the `virtual_networks` object. Can you do
```hcl
    vwan_security_configuration = optional(object({
      secure_internet_traffic = optional(bool, false)
      secure_private_traffic  = optional(bool, false)
    }), {})
```

Reply:
turns out you can :D